.github/workflows/trivy.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  push:
+    branches: ["*"]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    name: Trivy Scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Trivy
+        run: |
+          wget https://github.com/aquasecurity/trivy/releases/download/v0.38.0/trivy_0.38.0_Linux-64bit.deb
+          sudo dpkg -i trivy_0.38.0_Linux-64bit.deb
+      - name: Scan Image
+        run: |
+          trivy image --timeout 60m ghcr.io/containerd/nydus-snapshotter:latest
```

Comment:
How is the image `hcr.io/containerd/nydus-snapshotter:latest` built and uploaded?